primitives/src/lib.rs

```diff
@@ -139,7 +140,7 @@ pub mod issue {
 	}
 
 	// Due to a known bug in serde we need to specify how u128 is (de)serialized.
-	// See https://github.com/paritytech/substrate/issues/4641
+	// See https://github.com/paritytech/polkadot-sdk/issues/4641
```

Comment:
I think that the replacement here (and in two more occurrences in this file below) was not correct as this would now point at a different issue.

Reply:
You're correct. It was incorrectly replaced.